### PR TITLE
feat: add helm chart for CDN

### PR DESCRIPTION
**Pull request by fiam**
## Motivation and Context

- [ ] Tests or benchmark included
- [ ] Documentation is changed or added on [https://app.gitbook.com/](https://app.gitbook.com/)
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)

### CODE REVIEW

File: helm/cosmo/charts/cdn/values.yaml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    initialDelaySeconds: 10
+    timeoutSeconds: 5
+    failureThreshold: 5
+    periodSeconds: 10
+
+########################
+# Global configuration #
+########################
+
+#global:
```
Comment (StarpTech) [inline helm/cosmo/charts/cdn/values.yaml:136]: Remove

File: helm/cosmo/values.full.yaml [hunk 2]
```diff
@@ -79,6 +79,12 @@ ingress:
   #    hosts:
   #      - chart-example.local
 
+# Cosmo CDN
+cdn:
```
Comment (StarpTech) [inline helm/cosmo/values.full.yaml:83]: Please also add this to `values.yaml`

**Review by StarpTech — CHANGES_REQUESTED** [review]

**Review by StarpTech — APPROVED** [review]
LGTM

File: helm/cosmo/charts/cdn/values.yaml (hunk 1 above)
Comment (fiam) [reply helm/cosmo/charts/cdn/values.yaml]: Removed

**Pull request merged**